src/main/java/uk/ac/sanger/sccp/stan/service/operation/confirm/ConfirmSectionServiceImp.java

```diff
@@ -179,6 +186,14 @@ public ConfirmLabwareResult confirmLabware(User user, ConfirmSectionLabware csl,
                 measurements.add(new Measurement(null, "Thickness", String.valueOf(pa.getSampleThickness()),
                         sample.getId(), null, slot.getId()));
             }
+            if (!nullOrEmpty(sec.getRegion())) {
+                samplePositions.add(new SamplePosition(slot.getId(), sample.getId(), slotRegions.get(sec.getRegion()), null));
+            }
+            if (!nullOrEmpty(sec.getCommentIds())) {
+                for (Integer commentId : sec.getCommentIds()) {
+                    opComs.add(new OperationComment(null, commentIdMap.get(commentId), null, sample.getId(), slot.getId(), null));
```

Comment:
In cases where we are recording comments against each section, does it not not require region id ?

Reply:
The region is linked to a sample in a particular slot. The comment is also linked to the sample in a particular slot.

Reply:
Cool, Thank you 👍